spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -290,15 +290,15 @@
       let(:phone_confirmed) { false }
 
       it 'prompts to finish verifying profile, then redirects to SP' do
-        allow(FeatureManagement).to receive(:reveal_usps_code?).and_return(true)
-
+        allow(FeatureManagement).to receive(:reveal_usps_code?).and_return true
```

Comment:
Pretty sure the above line is required for the test to pass. If I remove it , it fails. Or is there a better way to do this?